docs/content/preview/migrate/macos.md

```diff
@@ -0,0 +1,27 @@
+<!--
++++
+private=true
++++
+-->
+
+Perform the following steps to install yb-voyager using brew for MacOS:
+
+1. [Tap](https://docs.brew.sh/Taps) to the `yugabyte` brew repository using the following command:
```

Comment:
Do we need to link the brew documentation for tap here? @amit-yb

Reply:
I included it just as a reference to a user (no assumptions that they may already know about it).